### PR TITLE
New package: uthash-1.9.9

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

File: srcpkgs/uthash/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'uthash'
+pkgname=uthash
+version=1.9.9
+revision=1
+short_desc="A hash table for C structures"
+maintainer="Diogo Leal <diogo@diogoleal.com>"
+license="CUSTOM"
```
Comment (xtraeme) [inline srcpkgs/uthash/template]: CUSTOM is not a valid license... in fact this software uses BSD license, iirc.

Comment (diogoleal) [inline srcpkgs/uthash/template]: Fixed

**Comment by Duncaen** [comment]
Sorry, a little bit late but why do you install `license.html`, in the root dir is a `LICENSE` file in plain text?

**Comment by pullmoll** [comment]
It's in the tarball as well. Would you fix it? I have a number of commits in the queue.

**Comment by Duncaen** [comment]
switched the file in 8e5745d

**Pull request merged**